virttest/libvirt_xml/vm_xml.py

```diff
@@ -1642,6 +1642,24 @@ def set_boot_order_by_target_dev(self, target, order):
                     device.boot = order
         self.set_devices(devices)
 
+    def set_boot_attrs_by_target_dev(self, target, **attrs):
```

Comment:
I assume set_boot_attrs_by_target_dev can cover previous set_boot_order_by_target_dev, do you want to deprecate it? Actually there are only 2 files ,which use set_boot_order_by_target_dev.

[root@chunfulap2 tp-libvirt]# find ./ -name "*.py" |xargs grep -i set_boot_order_by_target_dev
./libvirt/tests/src/bios/boot_integration.py:            vmxml.set_boot_order_by_target_dev(disk_target_dev, "1")
./libvirt/tests/src/bios/virsh_boot.py:        vmxml.set_boot_order_by_target_dev(target_dev, boot_order)

Reply:
@chunfuwen In terms of "don't break working code" I hadn't thought of removing the existing method. Do you think we should?

Reply:
From code maintainance pointview, Better have one single method existed. But as you mentioned, it may break working code. Therefore, I am fine with any choice you had.

Reply:
@chunfuwen Can you help merge?